operator/v1/types_klusterlet.go

```diff
@@ -162,6 +163,51 @@ type RegistrationConfiguration struct {
 	// +optional
 	// +kubebuilder:default:=100
 	KubeAPIBurst int32 `json:"kubeAPIBurst,omitempty"`
+
+	// PriorityBootstrapKubeconfigs defines the list of bootstrap kubeconfigs in the order of priority.
```

Comment:
Priority seems to indicate that there is a priority that needs to be set. But only order matters, right?

Reply:
Yes, order represent the priority of each bootstrapkubeconfig.